packages/react-components/src/components/ChatMessage/FluentChatMessageComponentWrapper.tsx

```diff
@@ -0,0 +1,281 @@
+import { ChatMessage } from '../../types';
+/* @conditional-compile-remove(data-loss-prevention) */
+import { BlockedMessage } from '../../types';
+import {
+  gutterWithAvatar,
+  gutterWithHiddenAvatar,
+  noMessageStatusStyle,
+  useChatMessageRenderStyles
+} from '../styles/MessageThread.styles';
+import { IPersona, PersonaSize, mergeStyles, Persona } from '@fluentui/react';
+import { mergeClasses } from '@fluentui/react-components';
+import { createStyleFromV8Style } from '../styles/v8StyleShim';
+import { MessageStatusIndicatorProps } from '../MessageStatusIndicator';
+import { ChatMessageComponentWrapperProps } from './ChatMessageComponentWrapper';
+import { ChatMessageComponent } from './ChatMessageComponent';
+import { ChatMessage as FluentChatMessage, ChatMyMessage as FluentChatMyMessage } from '@fluentui-contrib/react-chat';
+
+/**
+ * Props for {@link FluentChatMessageComponentWrapper}
+ *
+ * @private
+ */
+type FluentChatMessageComponentWrapperProps = ChatMessageComponentWrapperProps & {
+  message: ChatMessage | /* @conditional-compile-remove(data-loss-prevention) */ BlockedMessage;
+};
+
+/**
+ * The component for rendering a chat message using Fluent UI components
+ * and handling default and custom renderers.
+ * This component handles rendering for chat message body, avatar and message status.
+ * The chat message body, avatar and message status should be shown for both default and custom renderers.
+ *
+ * @private
+ */
+export const FluentChatMessageComponentWrapper = (props: FluentChatMessageComponentWrapperProps): JSX.Element => {
+  const {
+    message,
+    styles,
+    shouldOverlapAvatarAndMessage,
+    onRenderMessage,
+    onRenderAvatar,
+    showMessageStatus,
+    onRenderMessageStatus,
+    participantCount,
+    readCount,
+    onActionButtonClick,
+    /* @conditional-compile-remove(date-time-customization) */
+    onDisplayDateTimeString,
+    /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+    onFetchInlineAttachment,
+    /* @conditional-compile-remove(image-gallery) */
+    onInlineImageClicked,
+    /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+    inlineAttachments,
+    /* @conditional-compile-remove(mention) */
+    mentionOptions,
+    /* @conditional-compile-remove(file-sharing) */
+    fileDownloadHandler,
+    userId,
+    /* @conditional-compile-remove(file-sharing) */
+    onRenderFileDownloads,
+    defaultStatusRenderer,
+    statusToRender
+  } = props;
+  const chatMessageRenderStyles = useChatMessageRenderStyles();
+
+  const onRenderFileDownloadsMemo = useMemo(() => {
+    /* @conditional-compile-remove(file-sharing) */
+    return onRenderFileDownloads;
+    return undefined;
+  }, [/* @conditional-compile-remove(file-sharing) */ onRenderFileDownloads]);
+
+  // To rerender the defaultChatMessageRenderer if app running across days(every new day chat time stamp need to be regenerated)
```

Comment:
This comment doesn't seem to make much sense anymore, is there meant to be a prop that is time related in the dependency array?

Reply:
This code was just moved from MessageThread without much change. You are probably right that it can be removed

Reply:
Is there anything existing that updates the timestamp message from like "yesterday" to an actual date when a day elapses?

Reply:
Thanks for the clarifications! I'll add the dependency to return this change as it looks like it was missed for some time